features/analyze.feature

```diff
@@ -8,60 +8,130 @@ Feature: analyze
     With no options specified, performs analysis locally, and then sends the results to the Freshli web app so that the results can be viewed at at URL that will be provided in the command output.
 
 
-    NOTE: These scenarios are not complete nor properly tested because at the time of writing not everything is done. So the entire chain can't be tested yet.
+    NOTE: These scenarios are not complete the Web API is not implemented yet. So the results can't be verified yet.
```

Comment:
I would have thought that the contents of the cache database could be used to determine if the analysis was completed correctly or contained the correct results. The current version of the web API is essentially write-only, so it's not possible at this point in time to use it to validate that data collection has happened correctly.

Reply:
I want to have checks for the history stops, etc but I also wanted to raise the pull request to show some progress now that I have gotten past the bugs that were found while updating the tests.
I started working to figure out where/how I could get access to that information from an acceptance test - I was tracking through the code and thinking about console output, but had not thought to look in the database so I will try that! Thanks